web-app/HeimScripts/heatmap/run.R

```diff
@@ -301,6 +300,7 @@ buildFields <- function(df) {
   names(df)   <-
     c("PROBE","GENESYMBOL","SIGNIFICANCE","PATIENTID","VALUE")
   df["ZSCORE"] <- ZSCORE
+  #df[ is.na(df) ] <- 0  # All NAs to zero. Only for purpose of JSON
```

Comment:
I actually leave NA in JSON - this is intended. So that they can be handled properly on the frontend.